### DISCUSSION TITLE
App Layout with React?

### DISCUSSION THREAD

**Discussion by benash** [vmware/clarity#5661] [Q&A] [answered]
Hi, I am wondering if the [App Layout](https://clarity.design/foundation/app-layout/) features are available for React or web components. I only see examples for headers and navs in the Angular section. Thanks!

**Comment by bbogdanov** [comment, marked as answer]
Hi @benash,

At this stage the only way to accomplish `App Layout` is to combine the Angular and Core components.

`Vertical nav` and `header` are still missing in core. The `Vertical nav` on the other hand is under development and @hippee-lee can provide more details about it if needed.

**Comment by mathisscott** [comment]
@benash 
As @bbogdanov Mentioned, vertical nav is on the way. The header is also on the near-term roadmap for core. You will likely see you both sooner rather than later.

Issue #5627 can be followed if you want to track the status of the vertical nav.
